index.html

```diff
@@ -100,6 +100,9 @@ <h2>View States</h2>
         setupDoNotSellButton();
       };
 
+      // Add button for default view state
+      doc.innerHTML += `<button onClick="transcend.showConsentManager()">Default</button><br/><br/>`;
```

Comment:
To help us catch this in the future

Reply:
Can you help me understand why this helps prevent the issue?

Reply:
This issue happened because we made changes for custom viewstates without testing the default. This new line helps test the default/initial view state.